conftest.py

```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+
+this_dir = os.path.dirname(__file__)
```

Comment:
These three lines aren't needed :)

Reply:
but then the tests fail because they import fails
Is all the file not needed?

Reply:
ok did it :)